core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -56,7 +56,7 @@ private[spark] object Status {
     .createWithDefault(Int.MaxValue)
 
   val APP_STATUS_METRICS_ENABLED =
-    ConfigBuilder("spark.app.status.metrics.enabled")
+    ConfigBuilder("spark.metrics.appStatusSource.enabled")
```

Comment:
are these configs newly added in 3.0?

Reply:
The appStatus source is new in 3.0 [SPARK-25934]  
`spark.metrics.static.sources.enabled` parameter is also newly introduced for Spark 3.0 [SPARK-29654]  
The more delicate change in the proposal is about the potential replacement for `spark.sql.streaming.metricsEnabled`, that is already in use since a while [SPARK-17731]